### PR TITLE
Fix for xScale in multiChart

### PR DESCRIPTION
**Pull request by krmarien**
With this change the xScale can be set in the multiChart:

    var chart = nv.models.multiChart();
    chart.lines1.xScale(d3.time.scale());

Example: [http://plnkr.co/edit/TOcSRfEEp5YnnfiJ6Afd?p=preview](http://plnkr.co/edit/TOcSRfEEp5YnnfiJ6Afd?p=preview)

### CODE REVIEW

File: src/models/multiChart.js [hunk 1]
```diff
@@ -213,6 +213,9 @@ nv.models.multiChart = function() {
                 return a.map(function(aVal,i){return {x: aVal.x, y: aVal.y + b[i].y}})
             }).concat([{x:0, y:0}]) : [];
 
+            x = lines1.xScale();
```
Comment (liquidpele) [inline src/models/multiChart.js]: The x variable has stuff done to it above this, why would you overwrite it here?

Comment (krmarien) [inline src/models/multiChart.js]: X was initialized to a linear scale and that scale was applied to the xAxis. However changing it afterwards didn't work. 

So the initialization was removed and x is only assigned here and used afterwards. 

It is possible I missed something, but this was the only way I got a different xAxis scale working.

Comment (liquidpele) [inline src/models/multiChart.js]: yea, this doesn't seem right... this seems like it'll just throw away the:

    x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) { return d.x }))
                 .range([0, availableWidth]);		                 .range([0, availableWidth]);

And even then, you're not changing the xScale of anything but lines1.   I think what you probably want to do is add a new option on multiChart named xScale and have it then control the xScale method on all subcharts (or at least the ones that have that option).

Comment (jufemaiz) [inline src/models/multiChart.js]: "add a new option on multiChart named xScale and have it then control the xScale method on all subcharts"

YES! THIS IS EXACTLY RIGHT!

Sorry I've been tearing my hair out trying to get this work to allow for multilingual support and the D3 timeFormat.multi capability ><

Comment (Robsonsjre) [inline src/models/multiChart.js]: This is my try on solving this issue (#2039)
Its really simple, just add a new option xScale on your chart (like the lineChart)

**Comment by liquidpele** [comment]
Closing this as abandoned.

**Pull request closed without merging**